PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.webapp/src/com/microsoft/azuretools/webapp/ui/WebAppDeployDialog.java

```diff
@@ -189,6 +192,7 @@ public void widgetSelected(SelectionEvent e) {
             @Override
             public void widgetSelected(SelectionEvent e) {
                 sendTelemetry("DELETE");
+                TelemetryUtil.sendTelemetryOpEnd("delete app service", buildProperties());
```

Comment:
seems that "CREATE"->"open create webapp dialog" and "DELETE"->"delete app service", is there any traces for create app service